public/locales/en/translation.json

```diff
@@ -41,6 +41,7 @@
     "s3": "AWS S3",
     "slack": "Slack",
     "spectrum": "Spectrum",
+    "subreddit": "Subreddit",
```

Comment:
Fix the translation key (other languages as well) to `reddit` as it is referenced as `t('providers.reddit')`.